src/phoenix/db/database.py

```diff
@@ -78,6 +108,18 @@ def __init__(self, database: Optional[Path] = None) -> None:
         if int(cur.execute("PRAGMA user_version;").fetchone()[0]) < 1:
             cur.executescript(_INIT_DB)
 
+        engine = (
+            create_engine(f"sqlite:///{db_path}", echo=True)
+            if db_path
+            else create_engine(
+                "sqlite:///:memory:",
+                echo=True,
+                creator=_mem_db_creator,
+            )
+        )
```

Comment:
I think we are gonna want this to be declarative

Reply:
yea, agree